pycbc/io/hdf.py

```diff
@@ -935,8 +937,10 @@ def to_coinc_xml_object(self, file_name):
                 sngl.mchirp, _ = pnutils.mass1_mass2_to_mchirp_eta(
                         sngl.mass1, sngl.mass2)
                 sngl.eff_distance = (sngl.sigmasq)**0.5 / sngl.snr
-                sngl_combined_mchirp += sngl.mchirp
-                sngl_combined_mtot += sngl.mtotal
+                # If we ever decide to not do exact match, then mchirp
+                # needs combined from the different singles
```

Comment:
rewrite comment : 'If exact match is not used, take mean masses over different triggers'